CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## next
+
+### Added
+
+- New UI for executing queries in the NoSQL account. UI supports viewing results in a table, tree or a JSON view, showing the query stats and the RU cost.
+- New UI for executing search queries in the vCore account. UI supports viewing results in a table, tree or a JSON view.
+- New UI for creating, viewing and editing documents in the separated tab.
+- New UI supports themes from VS Code (except Monarch).
+
+- Jest for unit testing.
```

Comment:
CHANGELOG.md is rather customer facing (also for VS Marketplace), I don't think we need to mention these things here. Devs who are interested can review the Git history.

Reply:
Got it! Fixed